style.css

```diff
@@ -177,10 +195,20 @@ button {
   color: #fff;
   border: none;
   background-color: #7d73fc;
-  margin: 40px 0 0 0;
+  margin-top: 40px;
   padding: 10px 7px 10px 7px;
 }
 
+.f-contact h3 {
+  color: #9390ff;
+  font-family: Roboto, sans-serif, arial;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: -5px;
+}
```

Comment:
[REQUIRED] Please kindly make sure that your design matches the  [chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible. As a future web developer, you will have to build UI based on the design specified in the technical requirements, let's get used to it from the scratch :relaxed: 

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| ![Screenshot from 2022-02-16 14-17-54](https://user-images.githubusercontent.com/67608876/154233872-537f4b93-bdc8-4741-aeff-388ac1518e9d.png) | ![Screenshot from 2022-02-16 14-18-28](https://user-images.githubusercontent.com/67608876/154233920-42d34290-220b-4783-93cb-c141aedaa938.png) |

- [ ] In this particular case, you need to give your `email address` respective colour and it should be vertically centred along with your social icons